### PR TITLE
Removes all XCB related stuff from CMake. Rework X11 stuff.

### PR DESCRIPTION
**Pull request by luis-pereira**
We are not using it. They are just artifacts.

### CODE REVIEW

**Comment by palinek** [comment]
This actually reminds me to a discusssion in qlipper -> https://github.com/pvanek/qlipper/issues/34#issuecomment-248535444

Wouldn't it be feasible to add close requests for all the windows into the session end?
I mean, use something like `NETRootInfo(QX11Info::connection(), NET::CloseWindow).closeWindowRequest(wid);` [doc](https://api.kde.org/frameworks/kwindowsystem/html/classNETRootInfo.html#a9cdbb685d45dc467039f5d682110c176) (as in the panel's taskbar) for all (stil alive) windows into [the logout process](https://github.com/lxde/lxqt-session/blob/master/lxqt-session/src/lxqtmodman.cpp#L351)?

**Comment by luis-pereira** [comment]
@palinek It's probably feasible.
If we need it the future we should use FindXCB and drop pkg_check_modules() stuff.

File: lxqt-session/CMakeLists.txt [hunk 1]
```diff
@@ -73,7 +68,6 @@ target_link_libraries(lxqt-session
     Qt5::Widgets
     Qt5::DBus
     Qt5::X11Extras
-    ${XCB_LDFLAGS}
     X11
```
Comment (palinek) [inline lxqt-session/CMakeLists.txt]: Is the X11 still needed?

Comment (luis-pereira) [inline lxqt-session/CMakeLists.txt]: @palinek Turns out we need. We still need it for the numlock handling. I just updated the PR.

**Pull request merged**